### DISCUSSION TITLE
Share your proxy URL.

### DISCUSSION THREAD

**Discussion by Nu11Const** [acheong08/EdgeGPT#328] [Show and tell]
Full mirror URL also can share in here.

**Comment by riyazweb** [comment]
[acheong08](https://github.com/acheong08) hey please add compose version of edge chat also asap

**Reply by acheong08** [reply]
It's exam season. I can only add it when I actually have time

**Comment by riyazweb** [comment]
okay thank you for responding but dont forget it it dude it more better
than edge gpt chat version and all the best
